src/encoded/schemas/experiment_set.json

```diff
@@ -133,6 +133,9 @@
         "experiments_in_set.biosample.biosource.cell_line_tier": {
             "title": "Sample Tier"
         },
+        "experiments_in_set.biosample.biosample_category": {
+            "title": "Sample Category"
+        },
```

Comment:
would we want this facet in experiment collection view as well or no?

Reply:
We certainly could - do you think it would be useful?  The original motivation was for matrix construction but I guess it would be useful for searching experiments.

Reply:
I'm thinking about it, between Organism, Sample Type, and Biosource maybe another Biosource-related facet would be overkill. Easier to work with than the Biosource facet though because it has 100+ options. Maybe skip it for now and we can add it back later if we think it will be useful.